src/components/DataView/DataViewColumDrawer/DataViewColumnDrawer.styled.ts

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+
+export const ColumnTitle = styled.h2`
+  color: ${theme.newColors.almostBlack["100"]};
+  font-size: 20px;
+  font-weight: ${theme.fontWeight.normal};
+  line-height: 24px;
+  margin: 0 0 24px 0;
+`;
+
+export const StyledWrapper = styled.div`
+  display: flex;
+  margin-top: 30px;
+
+  & > .left {
+    border-right: 2px solid ${theme.newColors.grey2["100"]};
+    flex: 1;
+    padding: 0 32px 0 40px;
+    width: 265px;
+  }
+
+  & > .left .listItem {
+    margin: 3px 0px;
```

Comment:
I'm not sure what these 3px are for, also the checkbox list looks a bit to the right compared to the figma, this is because each checkbox has 10px padding for the hover, we can use -10px margin-left to align it.
```suggestion
    margin-left: -10px; 
```